src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.js

```diff
@@ -88,34 +81,38 @@ const convertInclude = async (includeBlock, ...restParams) => {
     return null;
   }
 
-  // This determines if the href has a leading '/'
-  const hrefTypePostion = () => (href.indexOf('/') === 0 ? 1 : 0);
-
-  // This checks if the supportedType is in the correct position of the href
-  const hrefIsSupported = () => supportedType =>
-    href.startsWith(supportedType, hrefTypePostion());
-
-  // This extracts the type from the href
-  const typeExtraction = Object.keys(supportedTypes).find(
-    hrefIsSupported(href),
-  );
+  const { includeType, classification } = includeClassifier({ href, pathname });
 
-  // This determines if the type is supported and returns the include type name
-  const includeType = supportedTypes[typeExtraction];
-  if (!includeType) {
+  if (
+    classification === 'not-supported' ||
+    classification === 'vj-include-not-supporting-amp'
```

Comment:
Just for my understanding. If we have a VJ include that doesn't support amp then we return `null` for canonical and amp? Or am I reading it wrong?

Reply:
In the case of a VJ include on canonical, it would be classified as `vj-include-canonical` and would its html would be fetched below: https://github.com/bbc/simorgh/pull/6873/files/d749bfe55b509c663511be75c1252f5412436506#diff-ace5f927a44cc0bc96a151479cba4bbaR106 

I think @Simon2828 is rolling` 'vj-include-not-supporting-amp'` into `'not-supported'` to make things a bit clearer